coreum/chain.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "coreum",
+  "status": "upcoming",
+  "network_type": "mainnet",
+  "website": "https://www.coreum.com",
+  "pretty_name": "Coreum",
+  "chain_id": "coreum-mainnet-1",
+  "bech32_prefix": "core",
```

Comment:
Please register this bech32 prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done: https://github.com/satoshilabs/slips/pull/1521

Reply:
@JeremyParish69 

The other PR has been merged